gpkit/nomials/math.py

```diff
@@ -519,6 +523,8 @@ def sens_from_dual(self, la, nu, result):  # pylint: disable=unused-argument
         self.relax_sensitivity = la
         if self.sgp_parent:
             self.sgp_parent.relax_sensitivity = la
+            if getattr(self.sgp_parent, "sgp_parent", None):
+                self.sgp_parent.sgp_parent.relax_sensitivity = la
```

Comment:
Why are there two layers of sgp_parent here?

Reply:
Oh let me guess, because you can have relaxations and SP approximations?

Reply:
yup.